blocksort.rs

```diff
@@ -1224,10 +1132,8 @@ unsafe extern "C" fn mainSort(
                         as Int32;
                     if hi > lo {
                         if verb >= 4 as libc::c_int {
-                            fprintf(
-                                stderr,
-                                b"        qsort [0x%x, 0x%x]   done %d   this %d\n\0" as *const u8
-                                    as *const libc::c_char,
+                            eprintln!(
+                                "        qsort [0x{:x}, 0x{:x}]   done {}   this {}",
```

Comment:
```suggestion
                                "        qsort [{:#x}, {:#x}]   done {}   this {}",
```
geeft dezelfde uitvoer.